Sources/AcmeSwift/Models/Account/AcmeAccountInfo.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
-import JWTKit
+@preconcurrency import JWTKit
```

Comment:
oh what's the issue with JWTKit here?

Reply:
I'm way out of my depth when it comes to all the sendable and concurrency stuff, but wouldn't it be safer overall to leave this as-is, assuming the JWTKit project will address the underlying issue at some point?

Reply:
This just silences the warnings in this project - it doesn’t change any of the underlying behavior.

That said, I realized my clone must be locked to v4 which is why I saw the warning - let me see if re-cloning brings me to v5 and gets rid of it!